examples/openvino/text-classification/configs/bert-base-jpqd.json

```
@@ -0,0 +1,47 @@
+[
+    {
+        "algorithm": "movement_sparsity",
+        "params": {
+            "warmup_start_epoch": 1,
+            "warmup_end_epoch": 2,
+            "importance_regularization_factor": 0.05,
+            "enable_structured_masking": true,
+        },
+        "sparse_structure_by_scopes": [
+            {"mode": "block", "sparse_factors": [32, 32], "target_scopes": "{re}.*BertAttention.*"},
+            {"mode": "per_dim", "axis": 0, "target_scopes": "{re}.*BertIntermediate.*"},
+            {"mode": "per_dim", "axis": 1, "target_scopes": "{re}.*BertOutput.*"},
```

Could it be possible to replace it with : 
```suggestion
            {"mode": "block", "sparse_factors": [32, 32], "target_scopes": "{re}.*Attention.*"},
            {"mode": "per_dim", "axis": 0, "target_scopes": "{re}.*Intermediate.*"},
            {"mode": "per_dim", "axis": 1, "target_scopes": "{re}.*Output.*"},
```
to have a more generic config (adapted to more architectures such as roberta for example)

---

@vuiseng9, can you please comment here?